README.adoc

```diff
@@ -163,6 +163,7 @@ spring.cloud.vault:
     scheme: http
     connection-timeout: 5000
     read-timeout: 15000
+    token: some-application-token
```

Comment:
Awesome!  I completely missed this documentation, thanks!  Might be worth noting that the 'authentication' key for authentication method is optional and seems to default to 'TOKEN'.